arbitrum-docs/run-arbitrum-node/data-availability-committees/03-deploy-mirror-das.md

```diff
@@ -239,11 +239,11 @@ Archive DA servers are servers that don't discard any data after expiring. Each
 
 To activate the "archive mode" in your DAS, set the parameter `discard-after-timeout` to `false` in your storage method. For example:
 
-```bash
+```shell
 --data-availability.s3-storage.discard-after-timeout=false
 ```
 
-Note that `local-file-storage` doesn't discard data after expiring, so the option `discard-after-timeout` is not available.
+Note that `local-file-storage` doesn't discard data after expiring by default, but expiration can be enabled with `enable-expiry` (the option `discard-after-timeout` is not available for this storage option).
```

Comment:
Same comment about the option name.

Reply:
Removed the mention to discard-after-timeout 💪